### ISSUE TITLE
Override Example/Docs

### ISSUE THREAD

**Issue by cretz** [rust-qt/cpp_to_rust#53]
Can you provide an example or docs for extending a Qt class? E.g. A simple example that extends `qt_widgets::main_window::MainWindow` and overrides [`closeEvent`](http://doc.qt.io/qt-5/qwidget.html#closeEvent)? This is commonly needed to intercept closing of a window.

**Comment by Riateche** [comment]
Yes, this is not implemented yet, but it's a high priority task.
